### PR TITLE
EREGCSC-607 EREGCSC-632 Regtext styling

### PR DESCRIPTION
**Pull request by whytheplatypus**
edge cases: 
- [x] paragraphs of the form (b)(1)

### CODE REVIEW

**Comment by whytheplatypus** [comment]
https://vxfdykx862.execute-api.us-east-1.amazonaws.com/dev-regtext-styling-may/433/Subpart-A/2020-11-06/

File: regulations/static/regulations/css/scss/partials/_reg_text.scss [hunk 1]
```diff
@@ -17,25 +17,54 @@
...
+    }
+
+    p {
+        margin: $spacer-3 0;
+        padding: 0 $spacer-3 0 $spacer-3;
+
+        @for $i from 3 through 10 {
+
+            &.depth-#{$i} {
+                padding-left: $spacer-3 + $spacer-3 * ($i - 2);
```
Comment (inkandsound) [inline regulations/static/regulations/css/scss/partials/_reg_text.scss]: I know I ended up changing this from `$spacer-3 + 16px` to conform to the design system stuff, but we could actually just write this as `$spacer-6` now that I look at it.

**Comment by inkandsound** [comment]
@whytheplatypus The code looks good. There's a small alignment issue when the paragraphs don't have citations like (a) in front of them though - they're indented a bit too much and getting `depth-3` when they should be `depth-1`:

https://vxfdykx862.execute-api.us-east-1.amazonaws.com/dev-regtext-styling-may/433/Subpart-A/2020-11-06/#433-32

Compare to eCFR: https://www.ecfr.gov/cgi-bin/text-idx?node=pt42.4.433&rgn=div5#se42.4.433_132

Everything else looks good to me! We may want to get Rebecca to take a look.

**Review by cgodwin1 — APPROVED** [review]

File: regulations/templates/regulations/partials/SECTION.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{% load render_nested %}
```
Comment (theabrad) [inline regulations/templates/regulations/partials/SECTION.html]: should be placed in subfolder for node_types

File: regulations/templates/regulations/partials/SECTION.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+{% comment %}
+    Used once per section. Contained by subparts at the top level.
+{% endcomment %}
+
+<section id="{{c.label | join:'-'}}" class="reg-section depth-{{c.label|length}}">
```
Comment (whytheplatypus) [inline regulations/templates/regulations/partials/SECTION.html]: sections don't need `depth-`

File: regulations/templates/regulations/partials/Paragraph.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{% load paragraphs %}
```
Comment (theabrad) [inline regulations/templates/regulations/partials/Paragraph.html]: Placed in sub folder node_types

File: regulations/templates/regulations/partials/SECTION.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    <h2 class="section-title" tabindex="0">
+    <a class="reference" href="#{{c.label | join:'-'}}"><i class="fa fa-link"></i></a>
+    {{c.title}} </h2>
+
+    <div class="paragraphs">
+        {{c.text | safe}}
+        {% for node in c.children %}
+            {% if node.node_type %}
+            {% render_nested "regulations/partials/"|add:node.node_type|add:".html" "regulations/partials/regulation_text.html" c=node first="false" %}
```
Comment (whytheplatypus) [inline regulations/templates/regulations/partials/SECTION.html]: can update children templates to use the node as the context

File: regulations/templates/regulations/partials/Citation.html [hunk 6]
```diff
@@ -0,0 +1,2 @@
+{% load citation %}
```
Comment (theabrad) [inline regulations/templates/regulations/partials/Citation.html]: Should be placed in node types subfolder

**Review by theabrad — CHANGES_REQUESTED** [review]

**Review by cgodwin1 — APPROVED** [review]

**Pull request merged**